code/modules/species/outsider/zombie.dm

```diff
@@ -115,17 +116,6 @@ GLOBAL_LIST_INIT(zombie_species, list(\
 	H.move_intent.move_delay = 6
 	H.stat = CONSCIOUS
 
-	if (H.wear_id)
-		qdel(H.wear_id)
-	if (H.gloves)
-		qdel(H.gloves)
-	if (H.head)
-		qdel(H.head) //Remove helmet so headshots aren't impossible
-	if (H.glasses)
-		qdel(H.glasses)
-	if (H.wear_mask)
-		qdel(H.wear_mask)
-	..()
```

Comment:
Don't remove `..()` here; this will cause wacky behaviour

Reply:
Did i fix it correctly?